Resources/Prototypes/_NF/Catalog/security_uplink_catalog.yml

```
@@ -1239,26 +1223,26 @@
         tags:
           - SecurityUplink
 
-# - type: listing
-  # id: UplinkSecurityNonLethalArmory
-  # name:  uplink-security-nonlethalarmory-name
-  # description: uplink-security-nonlethalarmory-desc
-  # productEntity: CrateArmoryNonLethalNfsd
-  # icon: { sprite: _NF/Structures/Storage/Crates/nfsd_secure1.rsi, state: icon }
-  # cost:
-    # FrontierUplinkCoin: 10
-  # categories:
-    # - UplinkSecurityUtility
-  # conditions:
-    # - !type:StoreWhitelistCondition
-      # whitelist:
-        # tags:
-          # - SecurityUplink
-    # - !type:BuyerJobCondition
-      # whitelist:
-        # - Bailiff
-        # - SeniorOfficer
-        # - Sheriff
+- type: listing
+  id: UplinkSecurityNonLethalArmory
+  name:  uplink-security-nonlethalarmory-name
+  description: uplink-security-nonlethalarmory-desc
+  productEntity: CrateArmoryNonLethalNfsd
+  icon: { sprite: _NF/Structures/Storage/Crates/nfsd_secure1.rsi, state: icon }
+  cost:
+    FrontierUplinkCoin: 10
+  categories:
+    - UplinkSecurityUtility
```

Put this in bundles?
```suggestion
    - UplinkSecurityBundles
```